src/Serilog.Sinks.Loggly/Properties/AssemblyInfo.cs

```diff
@@ -10,4 +10,14 @@
                                                        "6fe0fe83ef33c1080bf30690765bc6eb0df26ebfdf8f21670c64265b30db09f73a0dea5b3db4c9" +
```

Comment:
The attribute added below should replace this one (there's no `Serilog.Tests` in this solution - it's just a leftover from some early copy-pasta :-))